src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-22-01-13.mdx

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Have any PMMs looked at this? What's New is their domain so I want to make sure they know.

Reply:
yeah they sent me the text and i edited it.

Reply:
it's a very minor thing, to be honest. I don't think they'd care.

Reply:
but I'll check